src/commands/custom/deletemap.js

```diff
@@ -0,0 +1,82 @@
+const { SlashCommandBuilder, PermissionsBitField } = require("discord.js");
+const {
+    loadReactionMap,
+    gitCommit,
+    reactionMapPath,
+} = require("../../utils/reactionutils");
+const fs = require("fs");
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName("deletemap")
+        .setDescription("Delete a keyword from the reaction map")
+        .addStringOption((option) =>
+            option
+                .setName("keyword")
+                .setDescription("The keyword to delete")
+                .setRequired(true)
+        ),
+    async execute(interaction) {
+        const keywordToDelete = interaction.options.getString("keyword");
+
+        if (
+            !interaction.member.permissions.has(
+                PermissionsBitField.Flags.Administrator
+            )
+        ) {
+            return interaction.reply({
+                content: "You do not have permission to use this command.",
+                ephemeral: true,
+            });
+        }
```

Comment:
This can use the defaultPermissionFlag or whatever it is (see /clear) to not have to worry about the permissions